src/pages/valvonta-oikeellisuus/valvonta.svelte

```diff
@@ -39,6 +41,28 @@
 
   $: load(params);
 
+  // whoami -> versio -> id -> Future {energiatodistus: ET, korvattuEnergiatodistus: Maybe ET}
+  const fetchEnergiatodistus = (whoami, versio, id) => {
```

Comment:
tuntui helpommalta tehdä jonkin verran imperatiivisen kautta, mutta tarvittaessa voidaan kyllä jäsennellä funktionaalisemmaksikin

Reply:
joo kyllä tätä kannattaa vielä hioa, koska esim. maybe tyyppi näin käytettynä ei oikeastaan eroa nullista yhtään

Reply:
kokeile vielä vähän - tein itse tästä myös oman version